src/sentry/tasks/integrations.py

```diff
@@ -178,3 +182,65 @@ def migrate_repo(repo_id, integration_id, organization_id):
             integration,
             Organization.objects.get(id=organization_id),
         ).call()
+# Particulariliry For VSTS
+# TODO(lb): I want to say do this every 6 hours
+
+
+@instrumented_task(
+    name='sentry.tasks.integrations.kickoff_vsts_subscription_check',
+    queue='integrations',
+    default_retry_delay=60 * 5,  # TODO(lb): not sure what this should be
+    max_retries=5,
+)
+@retry()
+def kickoff_vsts_subscription_check():
+    organization_integrations = OrganizationIntegration.objects.filter(
+        integration__provider='vsts',
+        integration__status=ObjectStatus.VISIBLE,
+        status=ObjectStatus.VISIBLE,
+    ).select_related('integration')
+    six_hours_from_now = time() - mktime(timedelta(hours=6).timetuple())
+    for org_integration in organization_integrations:
+        organization_id = org_integration.organization_id
+        integration = org_integration.integration
+
+        try:
+            if 'subscription' not in integration.metadata or integration.metadata[
+                    'subscription']['check'] > six_hours_from_now:
+                continue
+        except KeyError:
+            pass
+
+        vsts_subscription_check.apply_async(
+            kwargs={
+                'integration': integration,
```

Comment:
This would be better as a simple base type (`int`, `string`, etc.) since it's going to be serialized and stored in a queue. It'll keep the disk space the queue needs to a minimum.

Using `integration_id`, and then looking that up in `vsts_subscription_check`, would probably be better.